torch/_higher_order_ops/map.py

```diff
@@ -13,8 +13,8 @@
 from torch._ops import HigherOrderOperator
 from torch._subclasses.fake_tensor import FakeTensorMode
 from torch._subclasses.functional_tensor import (
+    disable_functional_mode,
```

Comment:
Do you have some explanations for what was cleaned up and why? It's a bit hard to tell what improved

Reply:
Updated